ports/atmel-samd/boards/uchip/pins.c

```diff
@@ -1,6 +1,9 @@
 #include "shared-bindings/board/__init__.h"
 
 STATIC const mp_rom_map_elem_t board_global_dict_table[] = {
+    { MP_ROM_QSTR(MP_QSTR_LED_BUILTIN), MP_ROM_PTR(&pin_PA07) },
```

Comment:
For an onboard LED we usually just name it `LED`. I'd suggest switching to match other boards.

Reply:
alright, changed as circuitpython standard.
```suggestion
    { MP_ROM_QSTR(MP_QSTR_LED), MP_ROM_PTR(&pin_PA07) },
```